pyrasterframes/src/main/python/docs/raster-write.pymd

```diff
@@ -48,21 +50,47 @@ pandas_df.dtypes
 
 Viewing the DataFrame in Jupyter looks like this. 
 
-```python, evaluate=False
+```python show_pandas, evaluate=False
 pandas_df
 ```
 
 @@include[df-samples-output.md](static/df-samples-output.md)
 
-## PNG Overviews
-
-It is oftentimes convenient to have a simple overview image of the data, particularly if it spans a very wide area.
 
 ## GeoTIFFs
 
-GeoTIFF is one of the most common file formats for spatial data. 
+GeoTIFF is one of the most common file formats for spatial data, providing flexibility in data encoding, representation, and storage. RasterFrames provides a specialized Spark DataFrame writer for rendering a RasterFrame to a GeoTiff.
+
+One downside to GeoTIFF is that it is not a big data native format. To create a GeoTIFF all the data to be encoded has to be in the memory of one compute node (in Spark parlance, this is a "collect"), limiting it's maximum size substantially compared to that of a full cluster environment. When rendering GeoTIFFs in RasterFrames, you either need to specify the dimensions of the output raster, or be aware of how big the collected data will end up being.
+
+Fortunately, we can use the cluster computing capability to downlample the data into a more manageble saze. For sake of example, let's render a simple RGB overview image of our scene as a small raster:
```

Comment:
spelling "saze".

Reply:
and can we mention the downsampling method? I assume it's nearest nieghbor? Is it configurable?

Reply:
Will add statement on method used. Making it configurable:
https://github.com/locationtech/rasterframes/issues/222